docs/release_notes.rst

```diff
@@ -6,14 +6,19 @@ PUDL Release Notes
 v2024.X.x (2024-XX-XX)
 ---------------------------------------------------------------------------------------
 
-.. _release-v2024.8.0:
-
 Schema Changes
 ^^^^^^^^^^^^^^
 * Added :ref:`out_eia__yearly_assn_plant_parts_plant_gen` table. This table associates
   records from the :ref:`out_eia__yearly_plant_parts` with ``plant_gen`` records from
   that same plant parts table. See issue :issue:`3773` and PR :pr:`3774`.
 
+Bug Fixes
+^^^^^^^^^
+* Include more retiring generators in the net generation and fuel consumption
+  allocation. Thanks to :user:`grgmiller` for this contirbution :pr:`3690`.
+
```

Comment:
Non-blocking, could update this to specifically call out the inclusion of mid-year retirement cases.